source/contracts/reporting/Market.sol

```diff
@@ -440,6 +440,12 @@ contract Market is DelegationTarget, Typed, Initializable, Ownable, IMarket {
         return false;
     }
 
+    function isIndeterminate() public constant returns (bool) {
```

Comment:
Totally unrelated to this PR, but I wonder if we should have a modifier for market that is `onlyAfterFinalized`?  I suspect that `getFinalWinningReportingToken` does this check, but it would be nice to have the methods annotated to indicate when they can be called.  Similarly, I wonder if there is value in some other modifiers in `Market` like `onlyBeforeFinalized`?

Reply:
I'll put in a consider comment rather than implementing this only because we're battling the contract size currently :[